webpack.prod.config.babel.js

```diff
@@ -45,11 +45,14 @@ export default {
       allChunks: true,
     }),
     // optimizations
-    new webpack.optimize.UglifyJsPlugin({
-      sourceMap: true,
-      comments: false,
-      compress: {
-        drop_console: true,
+    new UglifyJsPlugin({
```

Comment:
Would be good to add a comment here saying not to make changes without busting the cache.